airbyte-integrations/connectors/source-holiday-api/README.md

```diff
@@ -0,0 +1,32 @@
+# Holiday API
+This directory contains the manifest-only connector for `source-holiday-api`.
+
+
+## Usage
+
+- You can use this connector as any other connector in Airbyte Marketplace.
+- You can load this connector in `pyairbyte` usinng `get_source`!
```

Comment:
@bnchrch a little typo in readme template

```suggestion
- You can load this connector in `pyairbyte` using `get_source`!
```

Reply:
Fixed: https://github.com/airbytehq/airbyte-platform-internal/pull/13505